core/src/components/picker-column-internal/picker-column-internal.tsx

```diff
@@ -401,15 +417,23 @@ export class PickerColumnInternal implements ComponentInterface {
     return (
       <Host
         tabindex={0}
+        role="radiogroup"
         class={createColorClasses(color, {
           [mode]: true,
           ['picker-column-active']: isActive,
           ['picker-column-numeric-input']: numericInput,
         })}
+        onFocus={this.onFocus}
       >
-        <div class="picker-item picker-item-empty">&nbsp;</div>
-        <div class="picker-item picker-item-empty">&nbsp;</div>
-        <div class="picker-item picker-item-empty">&nbsp;</div>
+        <div class="picker-item picker-item-empty" aria-hidden="true">
```

Comment:
The empty picker items are hidden from screen readers, otherwise they are read when the screen reader announces the entire section.